src/shared/classes/practiceable-base-page.ts

```diff
@@ -0,0 +1,60 @@
+import { Platform, NavController } from 'ionic-angular';
```

Comment:
Is the purpose of this new base page simply to house the practice mode banner?

Reply:
It's an abstract class so it can't house the banner html. It was so we only have to have the logic/selecting the data for checking if it's practice mode in once place instead of in every page of the app.

The reason for not using the Base Page is we have a number of pages which won't show the banner (Login, Journal, Candidate Details)

Reply:
By the end of this ticket there will be other things we will want to do across all the pages , which we should be able to do from here.